test/scripts/e2e-cp.sh

```diff
@@ -63,7 +63,7 @@ if [ "$OVN_GATEWAY_MODE" == "shared" ]; then
   if [ "$SKIPPED_TESTS" != "" ]; then
     SKIPPED_TESTS+="|"
   fi
-  SKIPPED_TESTS+="Should ensure load balancer service|[LGW]"
+  SKIPPED_TESTS+="Should ensure load balancer service|LGW"
```

Comment:
haha I would expect it to execute any tests with an L or a G or a W

Would `Should ensure load balancer service|\[LGW\]` work?

I guess the intention was to use LGW as a label and not match on tests descriptions that could through a LGW in randomly.

BTW, gingko does have native support for test labels we could start using instead.

Reply:
I think you meant *not* execute. Yes escaping would work. I think the intenion was to skip these tests for the egress service SNAT stuff:

```
e2e/egress_services.go:	ginkgotable.DescribeTable("[LGW] Should validate pods' egress uses node's IP when setting Network without SNAT",
e2e/egress_services.go:		ginkgotable.DescribeTable("[LGW] Should validate pods on different networks can reach different clients with same ip without SNAT"
```

However that is also the only place that has LGW tag, so it works fine.